### DISCUSSION TITLE
Version 1.27.0

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#518] [Announcements]
- Custom color schemes v2:
  - you can now have multiple custom color schemes
  - it is now possible to import and export color schemes
  - dynamic colors ("Material You") can be used in custom color schemes
  - new color scheme editor
- Tags that start with an emoji will now use that emoji in place of the default tag icon
- Added web search quick action
  - This action starts a web search in your default search app (e.g. the Google app). This replaces the default Google search action in new installation, but it has to be enabled manually on existing installations
- Added support for Android 13's per-app language feature
- Bug fixes and other optimizations

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.27.0'>Version 1.27.0</a>.</em>

**Comment by LitFill** [comment]
Can the default search app be changed to, for example, Chrome?

**Reply by MM2-0** [reply]
There are multiple apps that support the websearch action, for example Bing, and Firefox (but not Chrome, it looks like). If you have multiple of these apps installed, a chooser will be shown where you can select an app and whether you want to always use this app, or just this once. If no such chooser appears, it is possible that you either already have set a default app, or you have only one app installed.

That being said, you can always add a Google search websearch action to restore the previous behavior.

**Reply by LitFill** [reply]
thank you

**Comment by rahaaatul** [comment]
Thank you @MM2-0 !
